src/main/java/com/groupseven/sysc4806project/ShoppingCartService.java

```diff
@@ -0,0 +1,109 @@
+package com.groupseven.sysc4806project;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/api/carts")
+public class ShoppingCartService {
+
+
+    private final ShoppingCartRepository shoppingCartRepository;
```

Comment:
You're doing a mix of constructor injection and field injection here. Should probably stick to one style (IntelliJ suggests constructor injection)

Reply:
ah kk, will fix